src/bp/ui-shared/src/MainContent/Header/index.tsx

```diff
@@ -1,28 +1,55 @@
 import { Alignment, Button, Navbar, NavbarGroup, Tab, Tabs } from '@blueprintjs/core'
-import React, { FC } from 'react'
+import cx from 'classnames'
+import React, { FC, useState } from 'react'
+
+import MoreOptions from '../../MoreOptions'
 
 import style from './style.scss'
 import { HeaderProps } from './typings'
 
-const Header: FC<HeaderProps> = props => (
-  <Navbar className={style.header}>
-    {!!props.tabs?.length && (
-      <NavbarGroup>
-        <Tabs onChange={props.tabChange}>
-          {props.tabs.map(tab => (
-            <Tab key={tab.id} id={tab.id} title={tab.title} />
+const Header: FC<HeaderProps> = props => {
+  const [showingOption, setShowingOption] = useState()
+  return (
+    <Navbar className={cx(style.header, props.className)}>
+      {!!props.tabs?.length && (
+        <NavbarGroup>
+          <Tabs onChange={props.tabChange}>
+            {props.tabs.map(tab => (
+              <Tab key={tab.id} id={tab.id} title={tab.title} />
+            ))}
+          </Tabs>
+        </NavbarGroup>
+      )}
+      {!!props.buttons?.length && (
```

Comment:
Do you think it would make sense to add a "tab" property to buttons, so the header could handle automatically which buttons are displayed when you switch on another tab ? (thinking of flow/library)

Reply:
I was thinking of just changing the buttons list passed by the parent component so this one can stay "dumb" and lean, the logic for what's displayed in different tabs will already be handled in the parent anyway, wdyt?

Reply:
Either one would be good, but since we're going to use this pattern everywhere, we're gonna have to duplicate the logic on the parent to handle tab changes multiple times. Maybe if the "tab" property is there, it will handle it automatically, otherwise, it will act as a dumb component?

Reply:
We'll need to handle the tab change in the parent anyways as we need to display the right content 🤔 that's why I thought it was better to do all of it in the parent instead